pom.xml

```diff
@@ -59,9 +59,11 @@
 		<module>spring-cloud-aws-parameter-store</module>
 		<module>spring-cloud-aws-secrets-manager</module>
 		<module>spring-cloud-aws-ses</module>
+		<module>spring-cloud-aws-s3-parent</module>
```

Comment:
what's the purpose of `s3-parent` module? should be only `s3`?

Reply:
Ideally we would. I've added parent only to wrap actual `s3` module and another module for code generation. If we could add code generation to `s3` module, without polluting `s3` jar with classes responsible for code generation we could get rid of parent, but i haven't figured out how to do it. Suggestions welcome!